ChildProject/pipelines/samplers.py

```diff
@@ -645,7 +645,7 @@ class HighVolubilitySampler(Sampler):
     :type project: ChildProject.projects.ChildProject
     :param annotation_set: set of annotations to calculate volubility from.
     :type annotation_set: str
-    :param metric: the metric to evaluate high-volubility. should be any of 'awc', 'ctc', 'cvc'.
+    :param metric: the metric to evaluate high-volubility. should be any of 'words', 'turns', 'vocs'.
```

Comment:
hmm reading it like this I think LENA users will be confused. Perhaps words (awc), turns (ctc), vocs (cvc)?

Reply:
I tried to make it more clear in the docs (both the API and the cli)